system-test/datastore.ts

```diff
@@ -950,6 +963,235 @@ describe('Datastore', () => {
         });
       });
     });
+    describe('with a sum filter', () => {
+      it('should run a sum aggregation', async () => {
+        const q = datastore.createQuery('Character');
+        const aggregate = datastore
+          .createAggregationQuery(q)
+          .addAggregation(AggregateField.sum('appearances'));
+        const [results] = await datastore.runAggregationQuery(aggregate);
+        assert.deepStrictEqual(results, [{property_1: 187}]);
+      });
+      it('should run a sum aggregation with a list of aggregates', async () => {
+        const q = datastore.createQuery('Character');
+        const aggregate = datastore
+          .createAggregationQuery(q)
+          .addAggregations([
+            AggregateField.sum('appearances'),
```

Comment:
might be more interesting to include a sum, average, and a count all in one query

Reply:
I could do that in a separate block. This test is inside the `with a sum filter` block.

Reply:
Done